uvloop/includes/stdlib.pxi

```diff
@@ -146,8 +145,8 @@ cdef py_inf = float('inf')
 
 
 # Cython doesn't clean-up imported objects properly in Py3 mode,
-# so we delete refs to all modules manually (except sys)
-del asyncio, concurrent, collections, errno
+# so we delete refs to all modules manually (except sys, asyncio)
+del concurrent, collections, errno
```

Comment:
Just removed the `asyncio` to avoid losing the ability to call later the `asyncio.coroutines.debug_wrapper` when `uvloop` run on an interpreter different than 3.7. 

If you believe that this is a red flag, let's find another way to make the `_set_coroutine_debug ` ready to be executed by different Python versions.

Reply:
We need to keep `del asyncio`. Otherwise uvloop would hold refs to Tasks longer than necessary (that's Cython's fault and these `del`s are a workaround).

You can use `getattr`:

```cython
cdef aio_debug_wrapper = getattr(asyncio.coroutines, 'debug_wrapper', None)
```

And just check if it's not ``None`` before you call it.